### PR TITLE
bug/FP-1977: Remove scroll wheels from numeric input fields

### PR DESCRIPTION
**Pull request by edmondsgarrett**
## Overview
…

## Related

- [FP-1977](https://jira.tacc.utexas.edu/browse/FP-1977)
…

## Changes
- Added CSS for `input` to hide scroll arrows from browser
…

## Testing

1. Navigate to http://localhost:8000/register/request-to-submit/
2. Fill out all required fields prior to entity 'coverage estimates'
3. On each coverage estimate field, check that:
   - There are no arrows on the right side of the field
   - Step size is still enforced on the input (whole integers for the first two fields, step of 0.01 for last field)
4. Ensure form can be submitted successfully after all input validation met

## UI
Before:
17294E13-9417-4BFE-AD97-A04F4B8B3C60_1_102_o

After:
6DC720EB-C525-458A-88FF-E3EC29F90F88_1_102_o

…

### CODE REVIEW

File: apcd-cms/src/apps/registrations/static/registrations/css/submission_form.css [hunk 1]
```diff
@@ -56,3 +56,12 @@ input[name^="contact_phone"] {
     /* NOTE: A span 2 seems like enough, but only span 3+ does the job */
     grid-row: span 3;
 }
+
+/* Field Content */
+
+/* to remove 'scroll arrows' from numeric input fields */
+input::-webkit-outer-spin-button,
```
Comment (fnets) [inline apcd-cms/src/apps/registrations/static/registrations/css/submission_form.css:63]:
```suggestion
```

File: apcd-cms/src/apps/registrations/static/registrations/css/submission_form.css [hunk 2]
```diff
@@ -56,3 +56,12 @@ input[name^="contact_phone"] {
...
+input::-webkit-inner-spin-button {
+  -webkit-appearance: none;
+  margin: 0;
```
Comment (fnets) [inline apcd-cms/src/apps/registrations/static/registrations/css/submission_form.css:66]:
```suggestion
```

**Review by fnets — CHANGES_REQUESTED** [review]
Feel free to ignore, but I think there's just a little extraneous code in the CSS.

**Review by fnets — APPROVED** [review]

**Review by fnets — APPROVED** [review]
LGTM! A really simple solution

**Review by sophia-massie — APPROVED** [review]
LGTM!

**Pull request merged**